nrf_dm/doc/CHANGELOG.rst

```diff
@@ -9,6 +9,21 @@ Changelog
 
 All the notable changes to this project are documented in this file.
 
+main
+****
+
+All the notable changes added to the |NCS| main are documented in this section.
+
+Added
+=====
+* :c:func:`void nrf_dm_get_hopping_sequence(const nrf_dm_config_t * config, uint8_t hopping_sequence[NRF_DM_CHANNEL_MAP_LEN])`, to get the channel hopping sequence used during the measurement procedure.
```

Comment:
I think the build fails because of this.
Probably :c:func:`nrf_dm_get_hopping_sequence` should be used.

Reply:
https://github.com/nrfconnect/sdk-nrf/runs/8073180130?check_suite_focus=true

Reply:
Agreed.

Reply:
Thanks, giving it another try